intersphinx_registry/lookup.py

```diff
@@ -0,0 +1,20 @@
+import sys
+
+from intersphinx_registry import get_intersphinx_mapping
+from sphinx.ext.intersphinx import inspect_main
+
+if len(sys.argv) < 2:
+    sys.exit("Usage: python -m intersphinx_registry.lookup [packages...]")
+
+packages = set(sys.argv[1:])
```

Comment:
`set` will not respect order so it might be better to explicitly iterate?

Reply:
Yeah, but `get_intersphinx_mapping` takes a set and mypy complains otherwise. 

I don't think it will be used much with many packages, so I think we can have the order random for now and figure out if we want a stable order.

Reply:
It might actually be worth limiting it to one library at a time - if you had multiple `.inv` files in the same pager it might be harder to figure out which `:external` library to map back to. FWIW I think the `python -m sphinx.ext.intersphinx` lookup only supports one-at-a-time.